templates/stripes/stripe-text-image.php

```diff
@@ -1,13 +1,20 @@
-<?php if (!empty($image)) : ?>
-<div class="<?= $this->e($type, 'wrapperClasses') ?>"<?php
-    echo !empty($variation) ? " data-variation=\"{$variation}\"" : '';
-?>>
-    <div class="vssl-stripe-column<?= ($layout == 'image-left') ? ' image-left' : '' ?>">
-        <div class="vssl-stripe--text-image--text"><?= $text['html'] ?></div>
+<?php if (!empty($image)): ?>
+<div class="<?= $this->e($type, "wrapperClasses") ?>"<?php echo !empty(
+  $variation
+)
+  ? " data-variation=\"{$variation}\""
+  : ""; ?>>
+    <div class="vssl-stripe-column<?= $layout == "image-left"
+      ? " image-left"
+      : "" ?>">
+        <div class="vssl-stripe--text-image--text"><?= $text["html"] ?></div>
         <div class="vssl-stripe--text-image--image">
             <img
-              src="<?= $this->image($image, !empty($image_style) ? $image_style : null) ?>"
-              alt="<?= $image ?>"
+              src="<?= $this->image(
+                $image,
+                !empty($image_style) ? $image_style : null
+              ) ?>"
+              alt="<?= $image_alt ?? $image ?>"
```

Comment:
Using new $image_alt property for the alt attribute of the <img> tag, defaulting to the image filepath if not set.